### DISCUSSION TITLE
Extract tables randomly moves the bbox 20 pixels away

### DISCUSSION THREAD

**Discussion by merionum** [jsvine/pdfplumber#809] [Ask for help with specific PDFs]
Hi guys! Thanks for the awesome library!

I am doing a large parsing task and I noticed that for some of my documents I randomly encounter this problem where the bboxes are just moved away 20 pixels in both dimensions.

Could you help me to understand what goes wrong here please?

![image psd(1)](https://user-images.githubusercontent.com/21173351/217958635-0d3e2817-e6f1-4cdc-b915-80aaa9d364a0.png)

**Comment by merionum** [comment]
Attached the PDF

**Comment by jsvine** [comment]
Now answered in #810
